frontend/src/features/housingCompany/HousingCompanyDetailsPage.tsx

```diff
@@ -340,7 +345,14 @@ const LoadedHousingCompanyDetails = ({housingCompany}: {housingCompany: IHousing
                                             <div>{apartment.apartment_number}</div>
                                             <div>{apartment.floor}</div>
                                             <div>
-                                                {apartment.rooms} {(apartment.apartment_type as {value: string}).value}
+                                                {apartment.rooms}{" "}
+                                                {
+                                                    (
+                                                        apartment.apartment_type as {
+                                                            value: string;
+                                                        }
+                                                    ).value
+                                                }
```

Comment:
Do you have some different formatting settings, or why does this line get destroyed? 😅

(And also above the "Isännöitsijä" DetailField)

Reply:
Both were formatted via eslint-fix to look like that. This quoted part could actually fit within the line character limit so it is now "less exploded" - but the "Isännöitsijä" field has way fewer places to start a new line on combined with long variable names. It's forced to be formatted like that to fit the line char limit..